libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -373,10 +403,18 @@ export enum BucketInflightMethods {
   LIST = "list",
   /** `Bucket.delete` */
   DELETE = "delete",
-  /** `Bucket.putJson */
+  /** `Bucket.putJson` */
   PUT_JSON = "put_json",
-  /** `Bucket.getJson */
+  /** `Bucket.getJson` */
   GET_JSON = "get_json",
-  /** `Bucket.publicUrl */
+  /** `Bucket.publicUrl` */
   PUBLIC_URL = "public_url",
+  /** `Bucket.tryPut` */
+  TRY_PUT = "try_put",
```

Comment:
After your merge from `main`, you'll need to change these to camelCase:

```suggestion
  TRY_PUT = "tryPut",
```

Reply:
@revitalbarletz thanks, I saw #2468 but I thought it wasn't applying there.